apps/server/src/infra/calendar/service/calendar.service.spec.ts

```diff
@@ -91,4 +91,32 @@ describe('CalendarServiceSpec', () => {
 			);
 		});
 	});
+
+	describe('deleteEventsByScopeId', () => {
+		describe('when calling the delete events method', () => {
+			const setup = () => {
+				httpService.delete.mockReturnValue(
+					of(
+						axiosResponseFactory.build({
+							data: '',
+							status: HttpStatus.NO_CONTENT,
+							statusText: 'OK',
+						})
+					)
+				);
+			};
+
+			it('should call axios delete method', async () => {
+				setup();
+				await service.deleteEventsByScopeId('test');
+				expect(httpService.delete).toHaveBeenCalled();
+			});
+		});
+		it('should throw error if cannot delete a events', async () => {
```

Comment:
Please nest it in some outer `describe()` block to keep the "when ... should ..." convention.